base/src/org/compiere/model/MOrder.java

```diff
@@ -2187,6 +2187,12 @@ public boolean closeIt()
 			return false;
 		}
 
+		if (!calculateTaxTotal())
+		{
+			m_processMsg = "Error calculating tax";
```

Comment:
@nsarlabos Could you create messages in the dictionary so that the response to the user is multi-language?

`m_processMsg = "@ErrorCalculatingTax@";`